style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  background-color: #e5e5e5;
+  padding: 0;
+  margin: 0;
+}
+
+.hero {
+  background-image: url(assets/background-shapes.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  object-fit: contain;
+  max-width: fit-content;
+  width: 100%;
+  height: 718px;
+}
+
+main {
+  background-color: #fff|;
+  width: 100vw;
+  height: 100vh;
+  border-radius: 0 0 0 15%;
+}
+
+header {
+  padding-top: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav {
+  padding-right: 32px;
+}
+
+.my-logo a {
+  color: #6070ff;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 18px;
+  padding-left: 30px;
+  text-decoration: none;
+}
+
+.greetings {
+  color: #172b4d;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 32px;
+  margin-top: 140px;
+  padding-right: 34px;
+}
+
+.jobdescription {
+  color: #344563;
+  font-family: 'Poppins', arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 32px;
+  padding-right: 34px;
+}
+
+.connect-link {
+  color: #7f8cff;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 34px;
+}
+
+.social-media-links {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 16px;
+  list-style-type: none;
+}
```

Comment:
@hrusamaali Please make your website a replica of the The Figma design as required: 
![figma](https://github.com/hrusamaali/figma-portfolio-1/assets/122353030/8a00ab49-1364-4e28-9cb5-ac8e728bbeca)

**1. Your design:**
![Hafix-g](https://github.com/hrusamaali/figma-portfolio-1/assets/122353030/d563e685-b608-491b-bf1c-e7d95080cfc6)

**2. The Figma design:** 
![headline-mobile](https://github.com/hrusamaali/figma-portfolio-1/assets/122353030/9d5628f4-e27b-47c5-80be-e6ffbde1a67d)

It would be better if you follow the given requirement 🧑‍🍳